### PR TITLE
Updating Swedish language

### PR DESCRIPTION
**Pull request by Sopor**
## Please check that your pull request pass this checks:

-  [x] My changes respect the [Kodi add-on development rules](https://kodi.wiki/view/Add-on_rules)
-  [x] My code is both Python 2 and 3 compatible
-  [x] My code follows the code style of this project (PEP8, long lines allowed if make sense)
-  [x] I made sure there wasn't another Pull Request opened for the same update/change
-  [x] I have successfully tested my changes locally

## Types of changes

-  [ ] Bug fix (non-breaking change which fixes an issue)
-  [x] New feature (non-breaking change which adds functionality)
-  [ ] Breaking change (fix or feature that would cause existing functionality to change)
-  [ ] Adding a new region/hbo go version

### CODE REVIEW

File: resources/language/resource.language.sv_se/strings.po [hunk 1]
```diff
@@ -29,49 +29,49 @@ msgstr "FEL"
 #, fuzzy
```
Comment (arvvoid) [inline resources/language/resource.language.sv_se/strings.po]: Please remove the fuzzy flag

**Review by arvvoid — CHANGES_REQUESTED** [review]
Hello, thank you for your submission. 
Now that the translation is complete and checked just remove the #fuzzy flag from strings where present and I can merge this and include in a release.

File: resources/language/resource.language.sv_se/strings.po [hunk 2]
```diff
@@ -21,57 +21,55 @@ msgctxt "#30000"
...
 msgid "Unsupported region"
-msgstr ""
+msgstr "Regionen stöds inte"
 
 msgctxt "#30004"
 msgid "An unexpected error has occurred."
-msgstr ""
+msgstr "Ett oväntat fel har uppstått."
 
 msgctxt "#30005"
```
Comment (arvvoid) [inline resources/language/resource.language.sv_se/strings.po:40]: This string is untranslated

**Review by arvvoid — CHANGES_REQUESTED** [review]
Hello @Sopor, there was an additional fuzzy flag left, I removed it. Now all is ok, I just noticed one string left untranslated #30005

File: resources/language/resource.language.sv_se/strings.po (hunk 2 above)
Comment (Sopor) [reply resources/language/resource.language.sv_se/strings.po:40]: Sorry, but it is fixed now :)

**Review by arvvoid — APPROVED** [review]

**Comment by arvvoid** [comment]
@Sopor Now is ready for merging I just edited the last translator and revision date in the file

**Comment by arvvoid** [comment]
@all-contributors please add @Sopor for translation

**Pull request merged**